### PR TITLE
OSDOCS#4895: Installing a three-node vSphere cluster

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.13+

Issue:
This PR addresses [osdocs-4895](https://issues.redhat.com/browse/OSDOCS-4895)

Link to docs preview:

**vSphere**
- [Installing a three-node cluster on vSphere](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-three-node.html)
- Installing a cluster on vSphere with customizations > [Creating the installation configuration file](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html#installation-initializing_installing-vsphere-installer-provisioned-customizations) (note under step 2)
- Installing a cluster on vSphere with user-provisioned infrastructure > [Manually creating the installation configuration file](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-initializing-manual_installing-vsphere) (step 3)
- Installing a cluster on vSphere with user-provisioned infrastructure > [Creating the Kubernetes manifest and Ignition config files](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-user-infra-generate-k8s-manifest-ignition_installing-vsphere) (warning before step 3)
- Installing a cluster on vSphere with user-provisioned infrastructure > [Adding more compute machines to a cluster in vSphere](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#machine-vsphere-machines_installing-vsphere) (first note)

**VMC**

- [Installing a three-node cluster on VMC](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-three-node.html)
- Installing a cluster on VMC with customizations > [Creating the installation configuration file](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-customizations.html#installation-initializing_installing-vmc-customizations) (note under step 2)
- Installing a cluster on VMC with user-provisioned infrastructure > [Manually creating the installation configuration file](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-initializing-manual_installing-vmc-user-infra) (step 3)
- Installing a cluster on VMC with user-provisioned infrastructure > [Creating the Kubernetes manifest and Ignition config files](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#installation-user-infra-generate-k8s-manifest-ignition_installing-vmc-user-infra) (warning before step 3)
- Installing a cluster on VMC with user-provisioned infrastructure > [Adding more compute machines to a cluster in vSphere](https://56729--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-user-infra.html#machine-vsphere-machines_installing-vmc-user-infra) (first note)

QE review:
- [x] QE has approved this change.

Additional information:
Per the engineering installer [plan](https://docs.google.com/spreadsheets/d/1yl7p0IGQh0itLnoN1eJQWWtBq9zFdjGrM6mSyZVU-Ck/edit#gid=2101423022) this work is QE and docs only. SME approval is not required.

### CODE REVIEW

**Comment by mjpytlak** [comment]
@WenXinWei Three-cluster node on vSphere docs ready for QE review. PTAL. Thank you.

File: modules/installation-initializing-manual.adoc [hunk 1]
```diff
@@ -196,6 +202,10 @@ Make the following modifications:
 For more information about the parameters, see "Installation configuration parameters".
 endif::ash-default,ash-network[]
 
+ifdef::three-node-cluster[]
+. If you are installing a three-node cluster, modify the `install.config.yaml` file by setting the `compute.replicas` parameter to `0`. This ensures that the cluster's control planes are schedulable. For more information, see "Installing a three-node cluster on {platform}".
```
Comment (bscott-rh) [inline modules/installation-initializing-manual.adoc:206]:
```suggestion
. If you are installing a three-node cluster, modify the `install-config.yaml` file by setting the `compute.replicas` parameter to `0`. This ensures that the cluster's control planes are schedulable. For more information, see "Installing a three-node cluster on {platform}".
```

**Review by bscott-rh — APPROVED** [review]
Couple of minor suggestions, otherwise LGTM.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

**Pull request merged**